packages/web-components/tests/snapshots/dds-content-block-card-static.md

```diff
@@ -33,11 +33,12 @@
   <dds-button-group
     data-autoid="dds--button-group"
     role="list"
+    style="--dds--button-group--item-count:2;"
   >
     <dds-button-group-item
       data-autoid="dds--button-group-item"
       icon-layout=""
-      kind="primary"
+      kind="tertiary"
```

Comment:
@IgnacioBecerra just checking, would your change have caused this snapshot change?

Reply:
No, if anything it would modify the anchor within the shadow DOM but not any dds elements.